src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -81,6 +83,20 @@ class MoreInfoMediaPlayer extends LitElement {
               </mwc-button>
             `
           : ""}
+        ${!isUnavailableState(stateObj.state) &&
+        supportsFeature(stateObj, MediaPlayerEntityFeature.GROUPING)
+          ? html`
+              <mwc-button
+                .label=${this.hass.localize("ui.card.media_player.group")}
+                @click=${this._showGroupMediaPlayers}
+              >
+                <ha-svg-icon
+                  .path=${mdiSpeakerMultiple}
+                  slot="icon"
+                ></ha-svg-icon>
+              </mwc-button>
```

Comment:
```suggestion
              <ha-button
                .label=${this.hass.localize("ui.card.media_player.group")}
                @click=${this._showGroupMediaPlayers}
              >
                <ha-svg-icon
                  .path=${mdiSpeakerMultiple}
                  slot="icon"
                ></ha-svg-icon>
              </ha-button>
```

Reply:
This one is unresolved.

Reply:
The button above that one (the browse media one) is an `mwc-button`. Should I

1. Change just the addition as suggested
2. Change both (the other one in a separate commit?)
3. Leave as is

Reply:
You can change both if you want, but sooner or later we're going to change it anyway to a ha-button